conda/cli/main_list.py

```diff
@@ -217,7 +217,8 @@ def execute(args, parser):
     if args.canonical:
         format = 'canonical'
     elif args.export:
-        format = 'export'
+        print_explicit(prefix, args.md5)
```

Comment:
Basically I disable the -e (--export) since there is no channel info in the exported file. As a result, sometimes it may raise error when create a environment from file. 

--explicit option, on the other hand, will give comprehensive info about the environment, which is better for export and import behaviour.

I think some time we can bring back --export as long as we are confident on reproducing the env